DESIGN_CONSIDERATIONS.md

```diff
@@ -0,0 +1,77 @@
+## Optimize APIs and patterns for developers over designers
+
+Why?
+- There are thousands of them.  The cost of features and bugs is far greater.
+
+How?
+- Prefer readable and meaningful typed intellisense theme information, opposed to cryptic or unorganized theme information that requires lookups or design assistance to understand.
```

Comment:
not sure that we should mention Intellisense here, as we are referring to specific tool with that; `typed` and `discoverable types` should be enough